dashboard/src/components/Filter/CodeBlock.tsx

```diff
@@ -74,66 +205,37 @@ const CodeBlock = ({
-          className,
-        )}
-      >
-        <div
-          dangerouslySetInnerHTML={{ __html: highlightedCode.highlightedCode }}
-        ></div>
-      </pre>
-      {highlightedCode.highlightCount > 0 && (
-        <div className="py-4 pl-3">
-          <ul className={cn('flex gap-2 pb-2', highlightsClassname)}>
-            <li>
-              <span className="flex gap-1">
-                <span className="font-bold">
-                  <FormattedMessage
-                    id="codeBlock.highlights"
-                    defaultMessage={'Highlights:'}
-                  />
-                </span>
-              </span>
-            </li>
-            <li className="flex gap-1">
-              <ColoredCircle
-                quantity={highlightedCode.failCount}
-                backgroundClassName="bg-lightRed"
-              />
-              <FormattedMessage id="global.fails" defaultMessage={'Fails'} />
-            </li>
-            <li className="flex gap-1">
-              <ColoredCircle
-                quantity={highlightedCode.errorCount}
-                backgroundClassName="bg-orange-200"
-              />
-              <FormattedMessage id="global.errors" defaultMessage={'Errors'} />
-            </li>
-            <li className="flex gap-1">
-              <ColoredCircle
-                quantity={
-                  highlightedCode.highlightCount -
-                  highlightedCode.failCount -
-                  highlightedCode.errorCount
-                }
-                backgroundClassName="bg-mediumGray"
-              />
-              <FormattedMessage id="global.others" defaultMessage={'Others'} />
-            </li>
-          </ul>
-          <div className="flex items-center">
-            <LiaInfoCircleSolid />
-            <p className="pl-1 text-sm">
-              <FormattedMessage
-                id="codeBlock.highlightsTooltip"
-                defaultMessage="Teste"
-              />
-            </p>
-          </div>
-        </div>
-      )}
+      <div className="py-4 pl-3">
+        <span className="font-bold">
+          <FormattedMessage
+            id="buildAccordion.logExcerpt"
+            defaultMessage="Log Excerpt"
+          />
+        </span>
+      </div>
+
+      <CodeBlockDialog
+        CodeComponent={({ fullscreen }) => {
+          const className = fullscreen ? 'max-h-[520px]' : 'max-h-[425px]';
+
+          return (
+            <MemoizedCode
+              className={className}
+              highlightedCode={highlightedCode.highlightedCode}
+              footer={
+                highlightedCode.highlightCount > 0 ? (
+                  HighlightCounts({
+                    highlightedCode,
+                    highlightsClassnames: highlightsClassnames,
+                  })
+                ) : (
+                  <></>
+                )
+              }
+            />
+          );
+        }}
+      />
```

Comment:
This block is messy, we'll need to refactor it, if you can't do it right now, please create a ticket for it.
- `HighlightCounts` its a component but is being used as a function
- `CodeBlockDialog` should receive a onFullScreenToggle instead of receiving a component that than gets `fullscreen` 
- nit: create a const` footerElement = highlightedCode.highlightCount > 0 ? ...` instead of doing it on the return